_episodes/03-create.md

```diff
@@ -72,25 +72,25 @@ nothing to commit (create/copy files and use "git add" to track)
 
 > ## Places to Create Git Repositories
 >
-> Catsy starts a new project, `moons`, related to her `planets`
-> project.  Despite Luke's concerns, she enters the following
-> sequence of commands to create one Git repository inside another:
+> You start a new project, `csv`, related to the `metadata` project.
```

Comment:
So I prefer "you" here as you've written it, as I don't think we need to keep the Catsy thing going everywhere. I guess my only comment is we should try and keep that as consistent as we can.

Reply:
Welp sorry @chrissyrissmeyer

Reply:
No worries! Either way the Dracula is gone!